index.html

```diff
@@ -17,122 +17,154 @@
     <header>
       <nav>
         <a href="#home"> <img src="images/My-Logo.png" /> </a>
-        <img src="images/Union.svg" />
+        <img class="hamburger" src="images/Union.svg" />
+        <div class="nav-list">
+          <span>Portfolio</span>
+          <span>About</span>
+          <span>Contact</span>
+        </div>
       </nav>
     </header>
     <div class="container">
-      <section>
-        <div id="text1">
-          <h1>I’m Eddy Glad to see you!</h1>
+      <section id="intro">
+        <div id="content">
+          <div id="text1">
+            <h1>I’m Eddy <br />Glad to see you!</h1>
+          </div>
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me
+          </p>
+          <h4>LET’S CONNECT</h4>
+          <ul>
+            <li>
+              <a href="https://twitter.com/eddy56388406">
+                <img src="images/Vector.png" />
+              </a>
+            </li>
+            <li>
+              <a href="https://www.linkedin.com/in/pray-eddy-shimwa-18a6751b6/">
+                <img src="images/Linkedin icon.png" />
+              </a>
+            </li>
+            <li>
+              <a href="https://github.com/EddyShimwa">
+                <img src="images/Vector (1).png" />
+              </a>
+            </li>
+            <li>
+              <a href="N/A"> <img src="images/Vector (2).png" /> </a>
+            </li>
+            <li>
+              <a href="N/A"> <img src="images/Vector (3).png" /> </a>
+            </li>
+          </ul>
```

Comment:
- [ ] Great job working on the header section but I think you forget to add the background image ✅ so kindly take it from the Figma design and put it as your background image ✅📌🚀🔥

- Your design
![image](https://user-images.githubusercontent.com/58553711/200137564-5d043b1f-c18c-4862-9355-db7aa306b7d1.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/200137585-f518e254-7ab2-4f6c-803c-432125ef6076.png)